File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Kiểm tra số âm số dương/AMDUONG.cs
using System;
namespace Checkamduong
{
    public class Program
    {
        static void Main(string[]args)
        {
            int n = int.Parse(Console.ReadLine());
            if(n < 0)
                Console.Write("AM");
            else if(n == 0)
                Console.Write("KHONG");
            else
                Console.Write("DUONG");
        }
    }
}

File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Kiểm tra số âm số dương/AMDUONG.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;
    if(n < 0)
        cout << "AM";
    else if(n == 0)
        cout << "KHONG";
    else
        cout << "DUONG";
    return 0;
}